core/src/Serial/Kokkos_Serial_Parallel_Range.hpp

```diff
@@ -272,12 +272,13 @@ class ParallelScanWithTotal<FunctorType, Kokkos::RangePolicy<Traits...>,
   using Analysis =
       FunctorAnalysis<FunctorPatternInterface::SCAN, Policy, FunctorType>;
 
+  using value_type     = typename Analysis::value_type;
   using pointer_type   = typename Analysis::pointer_type;
   using reference_type = typename Analysis::reference_type;
 
   const FunctorType m_functor;
   const Policy m_policy;
-  ReturnType& m_returnvalue;
+  const pointer_type m_result_ptr;
```

Comment:
Since we do this, we no longer require ReturnType as template arg...we should remove it

Reply:
sry Idh4 was faster

Reply:
... and I answered the comment. Do you still want me to remove it temporarily?

Reply:
If we already decided that we will keep it to use it this way I am fine with it. Nevertheless I would be happier if we do not keep it around to just eventually use it.

Reply:
No, I guess it is fine in this case